0028-web-monetization/0028-web-monetization.md

```diff
@@ -139,13 +139,15 @@ Dispatched once the first ILP packet with a non-zero amount has been fulfilled b
 ```ts
 {
   detail: {
+    // Payment Setup Endpoint
     paymentPointer: String,
+    // Monetization ID
```

Comment:
I think the requestId here would need to change to reflect the new name monetization id right?

Reply:
I'm against renaming fields unless we absolutely have to

Reply:
If we don't rename it and keep it as monetization id doesn't that feel a little odd, maybe it should be called "Monetization Request ID". The issue is that if I see requestId and I search for request id I will not find Monetization ID in this doc. Ill only be able to find this comment to link the 2 together.

Reply:
Talked to Ben in person. Lets just keep this as it is in the PR with the Monetization ID comment.

Reply:
Also added Web-Monetization-ID header to the comment